packages/proxy/schema/model_list.json

```diff
@@ -4688,5 +4688,19 @@
     "reasoning": true,
     "max_input_tokens": 272000,
     "max_output_tokens": 128000
+  },
+  "accounts/fireworks/models/qwen3-coder-480b-a35b-instruct": {
```

Comment:
looks reasonable. if you don't mind showing a screenshot of a postman call to a local proxy to fireworks with these models working.